src/subscript/check_swatinit/pillarmodel.py

```diff
@@ -0,0 +1,300 @@
+import textwrap
```

Comment:
Move to tests, as it is not used by the script (as far as I could see)?

Reply:
Correct that `pillarmodel.py` is only used by tests for now. Having it deployed alongside check_swatinit allows usage elsewhere, while if it lies in tests, it cannot be used by others. Not sure about this.

Reply:
See the point - perhaps there should be common place for cross-script tools?

Reply:
Kept as is.